configure.ac

```diff
@@ -299,6 +299,9 @@ case "$host" in
 		CFLAGS="$CFLAGS -DPAGE_SIZE=4096 -DS_IWRITE=S_IWUSR"
 		CXXFLAGS="$CXXFLAGS -DPAGE_SIZE=4096 -DS_IWRITE=S_IWUSR"
 
+		# FIXME?  Only if __ANDROID_API__ < 24?
```

Comment:
An Android person should probably look at this - I don't know enough to sign off one way or another.

Reply:
If you read the Android headers, as of ABI 24, they have the `o` functions (`o` is for offset, vs. regular `long`).
`32` means `long`, which is 32 on 32bit, 64 on 64bit, and never widened on 32bit.
`o` stands a chance of being widened on 32bit.

Android's zlib seems stuck at `32` despite better being available.
Ultimately @grendello is likely to fixup our build after this PR to never use the zlib we carry.

It all affects the ability to use compressed files larger than 4gb.

Reply:
(@grendello on Android specifically; maybe we should ifdef android error in our zlib, now or in that)